geniza/corpus/models.py

```diff
@@ -13,3 +13,25 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+class LanguageScript(models.Model):
+    '''Combination language and script'''
+    language = models.CharField(max_length=255)
+    script = models.CharField(max_length=255)
+    display_name = models.CharField(max_length=255, blank=True, unique=True)
```

Comment:
You may need to regenerate the migration when you change that, since Django tracks metadata like that even though it doesn't affect the db. My preference would be to remove and regenerate the migration rather than make an additional one.

Reply:
OK, actually display name can't be unique because all the blank ones aren't unique! However, my information about not using null=true for char/text fields might be out of date. If we switch this to null = true then I think we can keep the unique constraint (which would be good).